crates/matrix-sdk/src/event_cache/mod.rs

```diff
@@ -496,6 +504,18 @@ impl EventCacheInner {
             }
         }
     }
+
+    async fn insert_or_update_relationship(&self, orig_ev_id: &EventId, related_ev_id: &EventId) {
+        let mut relations = self.relations.write().await;
+
+        if let Some(replacements) = relations.get_mut(orig_ev_id) {
+            replacements.insert(related_ev_id.to_owned());
+        } else {
+            relations
+                .insert(orig_ev_id.to_owned(), BTreeSet::from_iter(vec![related_ev_id.to_owned()]));
+        }
+        drop(relations);
```

Comment:
nit: no need to drop explicitly, it'll happen automatically at the end of the function's body.

Reply:
Yep, copy&paste issue. It's gone now.